assets/components/directDebit/directDebitPopUpForm/directDebitPopUpForm.scss

```diff
@@ -20,17 +20,42 @@
   position: relative;
   display: inline-block;
   top: 50%;
+  width: 345px;
+  padding: 10px 5px;
+  @include mq($from: mobileLandscape) {
+    width: 475px;
+    padding: 10px;
```

Comment:
For padding and margin, we try to use the horizontal and vertical layout variables. 

```scss
$gu-h-spacing: 20px;
$gu-v-spacing: 12px;
```

Reply:
I've been meaning to ask about that. I think it makes sense when they are directly related to that but a lot of the time it just makes it harder to work out what's going on eg. `padding: gu-h-spacing / 4;` seems needlessly opaque to me.

Reply:
Fair enough, the idea is to express everything in terms of an already set grid. I would only use it for straightforward usages for example `gu-h-spacing * 2`, etc.

Reply:
It's a fair point. We don't tend to stick to the grid as closely as dotcom because our pages tend to be bespoke landing pages and similar. That said, designs do typically tend to follow the `$gu-h-spacing` and `$gu-v-spacing` conventions for a lot of the padding/margins to keep the spacing consistent.

As a rule of thumb I don't tend to go lower than half, e.g. `$gu-v-spacing / 2`, or higher than maybe 4x, e.g. `$gu-h-spacing * 4`.